modules/crm_core_default_matching_engine/src/Plugin/crm_core_match/engine/DefaultMatchingEngine.php

```diff
@@ -27,7 +29,6 @@
  *   priority = 0,
  *   settings = {
```

Comment:
Removed for now. We don't have configurable plugins for now, so I suppose we can implement it when a use case appears.